runtime/cere/src/lib.rs

```diff
@@ -128,7 +128,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48001,
+	spec_version: 48002,
```

Comment:
If two minor digits are for testnet / devnet only and `spec_version` reflects the project version, can we bump the version to `4.9.0` and set `49000` here and in all the other runtimes in the [sister PR to dev](https://github.com/Cerebellum-Network/blockchain-node/pull/97)?

Reply:
Why is it for testnet / devnet only?
The is no v4.8.0 release so I can push the update to v4.8.0 and release it later